tests/unit/database/test_image.py

```diff
@@ -94,6 +95,14 @@ def runTest(self):
         self.assertEqual([image[1] for image in images],
                          ['image 1', 'image 2', 'image 3', 'image 4'])
 
+    def test_nonexistent_image_fetch(self):
+        """tries to retrieve the image id of an image that doesn't
```

Comment:
Ditto. Also, the sentences for a Multi-line docstrings are supposed to start on a new line and always end with a period. For example:

"""
Your multi-line
doc-string.
""""

More on that here: https://www.python.org/dev/peps/pep-0257/